filebeat/config/config.go

```diff
@@ -51,18 +51,20 @@ type Config struct {
 }
 
 type Registry struct {
-	Path         string        `config:"path"`
-	Permissions  os.FileMode   `config:"file_permissions"`
-	FlushTimeout time.Duration `config:"flush"`
-	MigrateFile  string        `config:"migrate_file"`
+	Path          string        `config:"path"`
+	Permissions   os.FileMode   `config:"file_permissions"`
+	FlushTimeout  time.Duration `config:"flush"`
+	CleanInterval time.Duration `config:"cleanup_interval"`
```

Comment:
By introducing `clean_interval` are we going to deprecate `clean_*` options in the `log` input later?

Reply:
this setting is a little different. With the new store cleanup is asynchronous. Instead of checking for old entries on every single update, we check periodically (even if the are no events). This setting configures the period (it is currently passed to the cursor-input).

The `clean_*` still need to exist, as they configure when an item is stale and can be removed.